demos/smart_classroom_demo/README.md

```diff
@@ -57,25 +64,57 @@ Options:
 
-Example of a valid command line to run the application:
+To run the demo, you can use public or pre-trained models. To download the pre-trained models, use the OpenVINO [Model Downloader](https://github.com/opencv/open_model_zoo/tree/2018/model_downloader) or go to [https://download.01.org/opencv/](https://download.01.org/opencv/).
+
+> **NOTE**: Before running the demo with a trained model, make sure the model is converted to the Inference Engine format (\*.xml + \*.bin) using the [Model Optimizer tool](https://software.intel.com/en-us/articles/OpenVINO-ModelOptimizer).
+
+Example of a valid command line to run the application with pre-trained models for recognizing students actions:
 ```sh
-./smart_classroom_demo -m_act <path to the person/action detection retail model .xml file> \
-                       -m_fd <path to the face detection retail model .xml file> \
-                       -m_reid <path to the face reidentification retail model .xml file> \
-                       -m_lm <path to the landmarks regression retail model .xml file> \
+./smart_classroom_demo -m_act <path_to_model>/person-detection-action-recognition-0005.xml \
+                       -m_fd <path_to_model>/face-reidentification-retail-0095.xml \
+                       -m_reid <path_to_model>/face-reidentification-retail-0095.xml \
+                       -m_lm <path_to_model>/landmarks-regression-retail-0009.xml \
+                       -fg <path_to_faces_gallery.json> \
+                       -i <path_to_video>
+```
+> **NOTE**: To recognize actions of students, use `person-detection-action-recognition-0005` model.
+
+Example of a valid command line to run the application for recognizing actions of a teacher:
+```sh
+./smart_classroom_demo -m_act <path_to_model>/person-detection-action-recognition-teacher-0002.xml \
+                       -m_fd <path_to_model>/face-reidentification-retail-0095.xml \
+                       -m_reid <path_to_model>/face-reidentification-retail-0095.xml \
+                       -m_lm <path_to_model>/landmarks-regression-retail-0009.xml \
                        -fg <path to faces_gallery.json> \
-                       -i <path to the input video>
+                       -teacher_id <ID of a teacher in the face gallery> \
+                       -i <path_to_video>
 ```
+> **NOTE**: To recognize actions of a teacher, use `person-detection-action-recognition-teacher-0002` model.
 
-> **NOTE**: Public Caffe\* models should be first converted to the Inference Engine format (`*.xml` + `*.bin`) using the [Model Optimizer tool](https://software.intel.com/en-us/articles/OpenVINO-ModelOptimizer).
+Example of a valid command line to run the application for recognizing first raised-hand students:
+```sh
+./smart_classroom_demo -m_act <path_to_model>/person-detection-raisinghand-recognition-0001.xml \
+                       -top_id <name of valid student action> \
```

Comment:
If `-top_id` has default value (as I proposed in another comment), setting this parameter is not required by default.

Reply:
Changed